oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -340,6 +342,21 @@ public APIGatewayProxyResponseEvent handleRequest(
 
                 LOG.info("Successfully processed request");
 
+                auditService.submitAuditEvent(
+                        OidcAuditableEvent.AUTH_CODE_ISSUED,
+                        clientSessionId,
+                        userSession.getSessionId(),
+                        clientId,
+                        userInfo.getSubject().getValue(),
+                        Objects.isNull(userSession.getEmailAddress())
+                                ? AuditService.UNKNOWN
+                                : userSession.getEmailAddress(),
+                        IpAddressHelper.extractIpAddress(input),
+                        Objects.isNull(userInfo.getPhoneNumber())
+                                ? AuditService.UNKNOWN
+                                : userInfo.getPhoneNumber(),
+                        persistentSessionId);
```

Comment:
Could you just remind me how come we dropped the metadata pairs? from the event as it currently exists in the Auth Code handler?: https://github.com/alphagov/di-authentication-api/blob/cf8b052285b36ae1a6605b30444f31d8e6d3e29b/oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthCodeHandler.java#L299

I'm not sure what they are used for, if anything, but they should be available as part of `userInfo.getClaim(<name>)` based on what the userinfo response will return: https://github.com/alphagov/di-authentication-api/blob/cf8b052285b36ae1a6605b30444f31d8e6d3e29b/auth-external-api/src/main/java/uk/gov/di/authentication/external/services/UserInfoService.java#L49

As you can see, `rp_pairwise_id` and `new_account` will always be part of that response. Internal subject ID will also be present, but this will be `userInfo.getSubject().getValue()` because it's set here: https://github.com/alphagov/di-authentication-api/blob/cf8b052285b36ae1a6605b30444f31d8e6d3e29b/auth-external-api/src/main/java/uk/gov/di/authentication/external/services/UserInfoService.java#L41

Sorry, I know we discussed this before, and I can't remember if you explained why we don't have these pairs any more, but in general, I think we should be trying to preserve the existing 'shape' of the audit event